pkg/health/clusterStatsHandler.go

```diff
@@ -285,6 +287,21 @@ func getStats(myid uint64, filterFunc func(string) bool, volumeField, countField
 
 	allVTableCounts := segwriter.GetVTableCountsForAll(myid)
 
+	// Read segment metadata file
+	smFile := writer.GetLocalSegmetaFName()
+	allSegMetas, err := writer.ReadSegmeta(smFile)
+	if err != nil {
+		log.Errorf("getStats: error when trying to read meta file=%+v. Error=%+v", smFile, err)
+		return nil, 0, 0, 0
+	}
+
+	// Create a map to store segment counts per index
+	segmentCounts := make(map[string]int)
+	for _, segMeta := range allSegMetas {
+		indexName := segMeta.VirtualTableName
+		segmentCounts[indexName]++
```

Comment:
on the first item, if `indexName` is not present in the map, this is going to crash